deploy/k8s/standalone-agent.yml

```diff
@@ -7,45 +7,422 @@ metadata:
+            metricsets:
+              - fsstat
+            period: 1m
+            processors:
+              - drop_event.when.regexp:
+                  system.fsstat.mount_point: ^/(sys|cgroup|proc|dev|etc|host|lib|snap)($|/)
+          - id: system/metrics-system.load-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.load
+              type: metrics
+            metricsets:
+              - load
+            condition: '${host.platform} != ''windows'''
+            period: 10s
+          - id: system/metrics-system.memory-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.memory
+              type: metrics
+            metricsets:
+              - memory
+            period: 10s
+          - id: system/metrics-system.network-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.network
+              type: metrics
+            metricsets:
+              - network
+            period: 10s
+            network.interfaces: null
+          - id: system/metrics-system.process-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.process
+              type: metrics
+            metricsets:
+              - process
+            period: 10s
+            process.include_top_n.by_cpu: 5
+            process.include_top_n.by_memory: 5
+            process.cmdline.cache.enabled: true
+            process.cgroups.enabled: false
+            process.include_cpu_ticks: false
+            processes:
+              - .*
+          - id: >-
+              system/metrics-system.process.summary-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.process.summary
+              type: metrics
+            metricsets:
+              - process_summary
+            period: 10s
+          - id: >-
+              system/metrics-system.socket_summary-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.socket_summary
+              type: metrics
+            metricsets:
+              - socket_summary
+            period: 10s
+          - id: system/metrics-system.uptime-095fd5f7-6aaf-4611-b527-640348d24457
+            data_stream:
+              dataset: system.uptime
+              type: metrics
+            metricsets:
+              - uptime
+            period: 10s
+      - id: e7db4bd9-4085-4775-aff6-7e6a46a23b88
+        name: cis_kubernetes_benchmark-1
+        revision: 1
+        type: cloudbeat
         use_output: default
-        logs: false
-        metrics: false
-      logging:
-        level: debug
+        meta:
+          package:
+            name: cis_kubernetes_benchmark
+            version: 0.0.1
+        data_stream:
+          namespace: default
+        streams:
+          - id: >-
+              cloudbeat-cis_kubernetes_benchmark.findings-e7db4bd9-4085-4775-aff6-7e6a46a23b88
```

Comment:
`cis_kubernetes_benchmark*` is deprecated, do you want to wait for the [migration](https://github.com/elastic/security-team/issues/3610) into `cloud_security_posture*`?

Reply:
A Todo with a gh issue link would be best

Reply:
Hey, the migration is being done at this moment. will be completed by the EOD.